website/docs/r/jwt_auth_backend.html.md

```diff
@@ -38,6 +38,8 @@ The following arguments are supported:
 
 * `jwt_validation_pubkeys` - (Optional) A list of PEM-encoded public keys to use to authenticate signatures locally. Cannot be used in combination with `oidc_discovery_url`
 
+* `jwt_supported_algs` - (Optional) A list of supported signing algorithms. Defaults to [RS256]
```

Comment:
Since it's being defaulted by Vault rather than by the Terraform Vault Provider, could we note that more explicitly here? Something like Vault 1.1.0 currently defaults to RS256 but future or past versions of Vault may differ?

Reply:
@tyrannosaurus-becks it's actually the default of the oidc dependency that go-oidc package.  The vault-plugin-jwt-auth package depends on that and uses its defaults by using an empty string slice.  Not sure if that changes your preference on language, but wanted to be clear here.

Reply:
Totally! Yes, I'm aware of that. I was thinking it'd be good to note that it ties to Vault because of how most people will be using the version of the plugin that's included in the Vault catalog at the time of a particular release. However, if you think it makes more sense to have the language refer to the plugin instead, that's fine with me.